app/views/spots/index.html.erb

```diff
@@ -5,7 +13,7 @@
   <tbody>
```

Comment:
Don't use a table when all you're doing is listing out a single row of items. `ul` is much more descriptive - also you have an empty `thead`, which is weird!